raiden-ts/src/transport/epics.ts

```diff
@@ -1503,3 +1520,352 @@ export const deliveredEpic = (
+            log,
+            latest$,
+            config$,
+          }).pipe(
+            tap((e) => log.debug('RTC: sent invite', callId, e)),
+            ignoreElements(),
+          ),
+        );
+      }),
+      take(1),
+      tap(() => log.info('RTC: got answer', callId)),
+      map((event) => {
+        connection.setRemoteDescription(new RTCSessionDescription(event.event.content.answer));
+        start$.next(null);
+        start$.complete();
+      }),
+      ignoreElements(),
+    ),
+    of(channel), // output created channel
+  );
+}
+
+// setup channel for callee
+function setupCalleeChannel$(
+  matrix: MatrixClient,
+  start$: Subject<null>,
+  info: CallInfo,
+  { httpTimeout }: RaidenConfig,
+  deps: Pick<RaidenEpicDeps, 'log' | 'latest$' | 'config$'>,
+) {
+  const { callId, peerId, peerAddress } = info;
+  const { log, latest$, config$ } = deps;
+  return fromEvent<MatrixEvent>(matrix, 'event').pipe(
+    filter(
+      // FIXME: remove any when MatrixEvent type exposes getAge & getContent methods
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      (event: any) =>
+        event.getType() === ('m.call.invite' as EventType) &&
+        event.getSender() === peerId &&
+        event.getContent()?.call_id === callId &&
+        event.getAge() <= (event.getContent()?.lifetime ?? httpTimeout) &&
+        !!event.getContent()?.offer,
+    ),
+    tap(() => log.info('RTC: got invite', callId)),
+    mergeMap((event) => {
+      // create connection only upon invite/offer
+      const connection = new RTCPeerConnection({
+        iceServers: [{ urls: 'stun:stun.l.google.com:19302' }],
+      });
+      connection.setRemoteDescription(new RTCSessionDescription(event.event.content.offer));
+      return merge(
+        // despite 'never' emitting, candidates$ have side-effects while/when subscribed
+        handleCandidates$(connection, matrix, start$, info, deps),
+        defer(() => connection.createAnswer()).pipe(
+          mergeMap((answer) => {
+            connection.setLocalDescription(answer);
+            const content = {
+              call_id: callId,
+              lifetime: httpTimeout,
+              version: 0,
+              answer,
+            };
+            // send answer
+            return waitMemberAndSend$(peerAddress, matrix, 'm.call.answer' as EventType, content, {
+              log,
+              latest$,
+              config$,
+            });
+          }),
+          tap((e) => {
+            log.debug('RTC: sent answer', callId, e);
+            start$.next(null);
+            start$.complete();
+          }),
+          ignoreElements(),
+        ),
+        fromEvent<RTCDataChannelEvent>(connection, 'datachannel').pipe(pluck('channel')),
+      );
+    }),
+    take(1),
+  );
+}
+
+// setup listeners & events for a channel, when it gets opened, and teardown when closed
+function listenChannel$(
```

Comment:
While reviewing the naming can get confusing: Do we mean payment channels or webrtc channels here? Might be good to be explicit about it

Reply:
I know this can get confusing, but this is how RTC channels are called. Just notice transport knows nothing about Raiden/on-chain channels, so this must be something else. Could you suggest better names than knowing these scopes?

Reply:
Not sure, in the spec of WebRTC they are called dataChannels I think, but that doesn't improve it much 😅 maybe simply `rtcChannel`?

Reply:
That would conflict with the action, which is called the same, but something in that direction could be useful.

Reply:
Done, gone with the name `dataChannel`. Thanks for the suggestion